specification/monitor/resource-manager/microsoft.insights/2016-03-01/logProfiles_API.json

```diff
@@ -57,7 +57,7 @@
           }
         },
         "x-ms-examples": {
-            "List alert rules": { "$ref": "./examples/deleteLogProfile.json" }
+            "Delete log profile": { "$ref": "./examples/deleteLogProfile.json" }
```

Comment:
seems like this example contains an additional response code that has not been modeled in swagger. Could you please verify? 

https://travis-ci.org/Azure/azure-rest-api-specs/jobs/258292847#L657